packages/bun-uws/src/HttpContext.h

```diff
@@ -38,6 +38,56 @@
 
 
 namespace uWS {
+
+namespace detail {
+
+template <typename T, typename... Args>
+auto makeArray(T&& el0, Args&&... values) {
+    return std::array<std::decay_t<T>, 1 + sizeof...(Args)>{
+        std::forward<T>(el0), std::forward<Args>(values)...
+    };
+}
+
+inline static const auto HTTP_METHODS = makeArray<std::string>(
```

Comment:
Are we losing something by making this no longer constexpr on the platforms that support it?

Reply:
Updated the PR to make everything constexpr